Library/Homebrew/test/sandbox_spec.rb

```diff
@@ -47,7 +48,7 @@
     end
 
     it "ignores bogus Python error" do
-      ENV["HOMEBREW_VERBOSE"] = "1"
```

Comment:
Same here: Wanna make sure it's okay to replace env var values with Homebrew.args stubs

Reply:
Yep 👍